lib/codegen/src/binemit/mod.rs

```diff
@@ -123,4 +123,18 @@ where
             emit_inst(func, inst, &mut divert, sink);
         }
     }
+
+    // output jump tables
+    for (jt, jt_data) in func.jump_tables.iter() {
+        let jt_offset = func.jt_offsets[jt];
+        for idx in 0..jt_data.len() {
+            match jt_data.get_entry(idx) {
+                Some(ebb) => {
+                    let rel_offset: i32 = func.offsets[ebb] as i32 - jt_offset as i32;
+                    sink.put4(rel_offset as u32)
+                }
+                None => sink.put4(0),
+            }
+        }
+    }
```

Comment:
I think this is a fine approach, although we should add a function to the `CodeSink` to notify it that the code has finished and have started on some read-only data. It may not mean a lot right now, although it will mean that we can make clif-util's disassembly view smart and only disassemble the bytes that are actually instructions :).